src/System.Text.RegularExpressions/tests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -34,10 +32,16 @@
     <Compile Include="Regex.Match.Tests.cs" />
     <Compile Include="Regex.Tests.Common.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetGroup)' == 'netstandard1.7'">
+    <Compile Include="PrecompiledRegexScenarioTest.cs" />
```

Comment:
Given that Caps and CapNames were deleted from the api, this test is not needed, instead Regexoption.Compiled tests need to be ported over. In fact, there should be compiler errors about missing Caps and CapNames apis..